recipe/meta.yaml

```diff
@@ -39,7 +39,8 @@ test:
   requires:
     - pip
     - pytest
-    - toolz {{ version }}
+    - toolz
```

Comment:
Maybe we could align on major & minor version, but allow patch version to float?

```suggestion
    - toolz ={{ ".".join(version|split(".")[:2]) }}
```

Reply:
I like that. Committed. Thanks!